docs/examples/modules/DropdownExample/index.js

```diff
@@ -19,12 +22,10 @@ export default [
         component: Dropdown,
       },
       {
-        title: 'Selection',
-        description: 'A dropdown can be used to select between choices in a form.',
-        component: Selection,
-      },
-      {
-        component: FriendSelection,
+        title: 'Dropdown opening direction',
+        info: 'By default, the dropdown menu opens downward, if the menu does not fit in the screen - it will open upward, but you can force set direction by specifying `direction` prop',
+        description: 'You can set the direction of opening dropdown menu',
+        component: DropdownDirection,
```

Comment:
Ideally, the docs should contain the same examples of the original website. Can you try to keep the same structure? If you think we need some examples that are Vue-only related, they should go to a "Usage" section at the bottom.